test/integration/annotation/modification_functions.go

```diff
@@ -143,3 +144,34 @@ func addAnnotation(t *testing.T, testRunner base.ClusterTestRunner) {
 		assert.Assert(t, err)
 	}
 }
+
+// debugAnnotatedResources prints current status for the exposed resources
+func debugAnnotatedResources(t *testing.T, testRunner base.ClusterTestRunner) {
+	pub, _ := testRunner.GetPublicContext(1)
+	prv, _ := testRunner.GetPrivateContext(1)
+
+	log.Printf("Debugging exposed resources (by annotation):")
+	i := 0
+	for _, cluster := range []*client.VanClient{pub.VanClient, prv.VanClient} {
+		// Retrieving the deployment
+		dep, _ := cluster.KubeClient.AppsV1().Deployments(cluster.Namespace).Get("nginx", v1.GetOptions{})
```

Comment:
Is it worth checking error returned on Get in case deployment is not there?

Reply:
Why not. I just added an extra assertion.